lib/inject.js

```diff
@@ -92,10 +92,18 @@ function injectAllRequirements(funcArtifact) {
     return BbPromise.resolve();
   }
 
-  this.serverless.cli.log('Injecting required Python packages to package...');
+  let injectProgress;
+  if (this.progress && this.log) {
+    injectProgress = this.progress.get('python-inject-requirements');
+    injectProgress.update('Injecting required Python packages to package');
+    this.log.info('Injecting required Python packages to package');
+  } else {
+    this.serverless.cli.log('Injecting required Python packages to package...');
+  }
 
+  let returnPromise;
   if (this.serverless.service.package.individually) {
-    return BbPromise.resolve(this.targetFuncs)
+    returnPromise = BbPromise.resolve(this.targetFuncs)
```

Comment:
Why not simply convert it to async function, and use `finally`?

Reply:
great suggestion